src/semantic/src/site/globals/site.overrides

```diff
@@ -22,6 +22,12 @@
   width: 100%;
 }
 
+#map-icon-admin {
+  position: absolute;
+  z-index: 4000;
+  padding: 1rem;
+}
+
```

Comment:
this is exactly the same as #map-icon-plus, i think we can make it a class and assign the same class name for map-icon-admin and map-icon-plus in order to avoid repeating ourselves

Reply:
done ;)